src/service/config.js

```diff
@@ -0,0 +1,12 @@
+const setMockService = (string) => {
```

Comment:
Tutaj jakies kongo sie odwalilo, czy nie wystarczy sama wartosc parametru ktory jest przekazywany jako string?
Powinno wystarczyc `import.meta.env.VITE_MOCKSERVICE`

Reply:
W vite jak się używa .env to wszystkie varki są przesyłane jako string. Musiałem dodać funkcję zaminiającą mój string na boolean. Dodam komentarz który to wyjaśni.